docs/en/administration/management/FE_configuration.md

```diff
@@ -1099,6 +1099,24 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - Description: The threshold of connection failures allowed for a blacklisted BE node. If a BE node is added to the BE Blacklist automatically, StarRocks will assess its connectivity and judge whether it can be removed from the BE Blacklist. Within `black_host_history_sec`, only if a blacklisted BE node has fewer connection failures than the threshold set in `black_host_connect_failures_within_time`, it can be removed from the BE Blacklist.
 - Introduced in: v3.3.0
 
+#### lock_manager_enabled
+
+- Default: false
+- Type: Boolean
+- Unit: -
+- Is mutable: No
+- Description: Whether to enable the lock manager. The lock manager performs central management for locks. For example, it can control whether to refine the granularity of metadata locks from the database level to the table level.
```

Comment:
When it's `true`, it can further control whether ...

otherwise, it's not clear enough. Just a suggestion.